g11n-ws/tools/tool-trans-fetcher/src/main/java/com/vmware/vip/fetcher/monitor/UpdateThread.java

```diff
@@ -12,17 +12,17 @@
  * At present, this code is not used
  */
 public class UpdateThread extends Thread {
-
-    @SuppressWarnings("static-access")
+    private static boolean flag = true;
     synchronized public void run() {
         MultiComponentsDTO syncObject = new MultiComponentsDTO();
         synchronized (syncObject) {
-            boolean flag = true;
+           
             while (flag) {
```

Comment:
use 'true' directly here.

Reply:
use the 'true' directly is not recommend in java program

Reply:
flag is unnecessary

Reply:
no, use the multi-thread, not recommend  use direct 'true' in run method